docs/gitbook/recommend/movielens_cf.md

```diff
@@ -61,7 +61,8 @@ partial_result as ( -- launch DIMSUM in a MapReduce fashion
     movie_features f
   left outer join movie_magnitude m
 ),
-similarity as ( -- reduce (i.e., sum up) mappers' partial results
+similarity as (
+    -- reduce (i.e., sum up) mappers' partial results 
```

Comment:
Is it possible to use `'` is comments in Hive v2.3.0?

Reply:
Yes.

Reply:
If the number of `'` is even in comment, `'` can be used in the same line:

```
> select (1 --'sample'
> );
```